backend/integrationTesting/integrationSuite.go

```diff
@@ -13,11 +13,16 @@ type IntegrationTestSuite struct {
 }
 
 func (suite *IntegrationTestSuite) SetupSuite() {
+	/** Since the  testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> Ww make sure it's not used in parallel using a mutex */
```

Comment:
```suggestion
	/** Since the  testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> We make sure it's not used in parallel using a mutex */
```